lib/spack/spack/cmd/style.py

```diff
@@ -47,6 +48,7 @@ def grouper(iterable, n, fillvalue=None):
 #: order in which tools should be run. flake8 is last so that it can
 #: double-check the results of other tools (if, e.g., --fix was provided)
 tool_order = ["isort", "mypy", "black", "flake8"]
+tool_versions = {"isort": "4.3.5", "mypy": "0.900:"}
```

Comment:
```suggestion
tool_versions = {"isort": "4.3.5:", "mypy": "0.900:"}
```

Reply:
Is that okay with @tgamblin? He mentioned doing 4.3.5 explicitly above.

Reply:
oh I see it's a bug, so looks like we should try that.

Reply:
The 4.3.5 was rather arbitrary. It was the first version that supports `pyproject.toml` for configuration. No idea if we use any features that were added in later versions or not.

Reply:
LGTM!